programs/zstdless

```diff
@@ -1,2 +1,6 @@
 #!/bin/sh
-zstdcat "$@" | less
+
+# TODO: Address quirks and bugs tied to old versions of less, provide a mechanism to pass flags directly to zstd
+
+export LESSOPEN="|zstd -cdfq %s"
```

Comment:
I think this should be

```
export LESSOPEN="|-zstd -cdfq %s"
```

I.e., we should add a dash so that it's also invoked on stdin (e.g., `... | zstd | zstdless`).